operator/controllers/reconcilers/server/server_reconciler.go

```diff
@@ -104,14 +104,19 @@ func (s *ServerReconciler) Reconcile() error {
 	return nil
 }
 
-func updateCapabilities(extraCapabilities []string, podSpec *v1.PodSpec) {
-	if len(extraCapabilities) > 0 {
+func updateCapabilities(capabilities []string, extraCapabilities []string, podSpec *v1.PodSpec) {
+	if len(extraCapabilities) > 0 || len(capabilities) > 0 {
 		for _, container := range podSpec.Containers {
 			for idx, envVar := range container.Env {
 				if envVar.Name == EnvVarNameCapabilities {
-					capabilitiesStr := strings.Join(extraCapabilities, ",")
-					val := fmt.Sprintf("%s,%s", envVar.Value, capabilitiesStr)
-					container.Env[idx] = v1.EnvVar{Name: EnvVarNameCapabilities, Value: val}
+					if len(capabilities) > 0 {
```

Comment:
maybe add a note in code and docs that `capabilities` is going to take precedence?

Reply:
Fixed docs in https://github.com/SeldonIO/seldon-core/pull/4975/commits/3ebe7f94dbd9449c9d9a6463e4b70c9ddfff3aa0
Also removed deprecated comment for now.